### ISSUE TITLE
Unicode in Cote

### ISSUE THREAD

**Issue by AnasGhrab** [Parisson/Telemeta#23]
If a special Unicode letter is used in the code field, at least in the media_fonds table, like ā or an arabic letter, its gives a problem : 

File "/usr/local/lib/python2.7/dist-packages/django/core/urlresolvers.py", line 456, in _reverse_with_prefix
   (lookup_view_s, args, kwargs, len(patterns), patterns))

NoReverseMatch: Reverse for 'telemeta-resource-detail' with arguments '(u'fonds', u'\u0634\u0645\u0647\u0640\u0632\u062e\u0639\u0633\u0634\u0648\u0642\u0634')' and keyword arguments '{}' not found. 1 pattern(s) tried: ['archives/(?P<type>[A-Za-z0-9._-]+)/(?P<public_id>[A-Za-z0-9._-]+)/$']

**Comment by yomguy** [comment]
Yes, the only possible characters for the code are A-Za-z0-9.- since these chars will be in the URLs. We could add some more, but I'm afraid no *very* special chars could be used there. What are yours?

**Comment by AnasGhrab** [comment]
I think it's not necessary to put any special character there, but it's import to make a check that it does not contain any before saving the notice, otherwise we have a 500 error..

**Comment by yomguy** [comment]
Oh you're right, we should only get a message error in the forms..
